en/Plugins/Unique note creator.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
How are you renaming files, @marcusolsson ?  I'm used to git picking up that a file has been renamed (instead of deleted and then re-added) - in which case it shows in the new file the diffs against the old one.

Here, it's impossible to know what edits are made, at least from the Web UI.

Reply:
Thanks. I've not noticed that on Mac. I wonder whether the JetBrains UIs do something special to associate old and new files together in the commit...